dask_sql/context.py

```diff
@@ -829,7 +848,7 @@ def _register_callable(
 
             elif schema.functions[lower_name] != f:
                 raise ValueError(
-                    "Registering different functions with the same name is not allowed"
+                    'Registering multiple functions with the same name is only permitted when the parameter "replace=True" is used.'
```

Comment:
Think we should be good without quotes around the keyword argument here:

```suggestion
                    "Registering multiple functions with the same name is only permitted if replace=True"
```